### ISSUE TITLE
Feature: Print an SD sequence

### ISSUE THREAD

**Issue by mpogue2** [mpogue2/SquareDesk#278]
Code is at line 8388, in on_ActionFilePrint_triggered().  It would be nice to be able to print the whole thing with the nice images, now that we have them from Dan's large SD commit.

**Comment by danlyke** [comment]
2b0456c08d33f8ea490058df59aa26059d4b7bb5 uses the same code as copy to/save as HTML, but unfortunately the HTML conversion that we use for the print doesn't know what to do with the SVG tag, so we're going to have to do something else there.

Maybe we can generate PNGs and include them as inline <img> tags? Or maybe we actually have to figure out how to do full-on print formatting (ugh).

**Comment by mpogue2** [comment]
@danlyke This seems to work fine for me on Mac OS X!  Exactly as I would expect, I just did a Save As HTML..., and this is what I see:
![sd_sequence](https://user-images.githubusercontent.com/6316003/35773574-28c9266c-090b-11e8-8d82-22d618e1454e.png)

Maybe you are having a Linux-specific problem?

**Comment by mpogue2** [comment]
Oh never mind.  It's the PRINT that's the problem.  This is what I get when doing a Print SD Sequence:

![sd_sequence_print](https://user-images.githubusercontent.com/6316003/35773588-8df6c986-090b-11e8-808f-4fb002702883.png)

**Comment by mpogue2** [comment]
I tried for several hours to make QWebEngineView->print() work, with no success.  I was able to set it via setHtml(), but when I told it to print(), I always got a blank page.  The print async call did return true, so some bit of code thought everything was fine.

I'm giving up for now on that approach.  Temporary workaround is to save it as HTML, then open a browser and print from there.  Yes, that's quite ugly.

**Comment by danlyke** [comment]
Yeah, the current print output is just the straight text formations because the parser setHtml there was giving me weird garbage for the <svg> tag contents. File under "okay for now, but..."
